js/controller.js

```diff
@@ -1390,6 +1408,27 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.renderSkin();
     },
 
+    getAdRemainingTime: function() {
+      var remainingTime = 0;
+
+      var isLive = (this.state.currentAdsInfo.currentAdItem) ? this.state.currentAdsInfo.currentAdItem.isLive : false;
+      var isSSAI = (this.state.currentAdsInfo.currentAdItem) ? this.state.currentAdsInfo.currentAdItem.ssai : false;
+
+      if (isLive) {
+        remainingTime = parseInt((this.state.adStartTime + this.state.adVideoDuration * 1000 - Date.now()) / 1000);
```

Comment:
Is the adStartTime in ms but the adVideoDuration in seconds? Just double checking

Reply:
Changed adVideoDuration to ms.